src/plopp/model.py

```diff
@@ -56,10 +56,10 @@ def input_node(obj):
 
 def widget_node(widget):
     n = Node(func=lambda: widget.value)
-    # TODO: Our custom widgets have a 'watch' method instead of 'observe' because
-    # inheriting from VBox causes errors when overriding the 'observe' method
+    # TODO: Our custom widgets have a '_plopp_observe' method instead of 'observe'
```

Comment:
I think people also put `_` at the end for API methods. After all, this is not "private", but intended to be overridden?